python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_base_group_chat.py

```diff
@@ -44,7 +44,10 @@ class BaseGroupChat(Team, ABC, ComponentBase[BaseModel]):
 
     def __init__(
         self,
-        participants: List[ChatAgent],
+        *,
+        name: str,
+        description: str,
+        participants: List[ChatAgent | Team],
```

Comment:
Nice! 
Sounds like a good name for this PR might be

"Supporting Teams as Participants in a GroupChat".

- You can now pass in a team as a participant (in addition to agents)
- If a team is passed in .. it runs till execution and the TeamResult is _added to conversation history_ (or the right description of exactly what happens)


Overall, very exciting! Thanks @ekzhu !!

Reply:
You are so fast! I was just writing the description. It is updated now.